app/models/oauth_access_token.rb

```diff
@@ -0,0 +1,15 @@
+# typed: strict
+class OauthAccessToken < ApplicationRecord
+  # Associations
+  belongs_to :user, foreign_key: :user_id
+  belongs_to :oauth_application, foreign_key: :application_id
```

Comment:
Consider adding a database index on the 'user_id' and 'application_id' columns to improve query performance, especially if these associations are frequently queried.